### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by JeronGibson** [webdevnerdstuff/amazon-smile-redirect#115] [General]
Thank you for Amazon Smile Redirect.
I appreciate your efforts to make it easier for me to contribute to my chosen charity.
Your code is responsible for most of my donations, without it I would not have remembered to use the Smile URL when shopping.

I am sad to see the Amazon Smile program end.  The reasoning that it had too many charities, and was too broad, is exactly why it was good.  Small charities need small donations and they will be overlooked by Amazons decision to make donations to only large programs.

**Comment by webdevnerdstuff** [comment]
Thank you for your kind words (and donation), I really do appreciate it.

**Comment by Naton-1** [comment]
Hear hear! This extension has been a blessing for me as well. Truly, thank you for creating this and publishing it for the world to use. It was good while it lasted :)

**Comment by nuenuewei** [comment]
This extension solved a minor, but very frequent, annoyance for me; thousands of times over many years. It helped me make a small difference in the world as well. Removing it feels kind of sad. Thank you very much for your work.
